README.md

```diff
@@ -170,6 +170,8 @@ Comming soon. We are contributing to support this now. See: https://github.com/f
 
 Wayland compositor (weston) must be running before running the program when you use the Wayland backend.
 
+The DRM device file name must be set to the environment variable `FLUTTER_DRM_DEVICE` when you use the DRM backend. For example, run `export FLUTTER_DRM_DEVICE="/dev/dri/card1"`. If it is not set, `/dev/dri/card0` is used.
```

Comment:
Note that `FLUTTER_DRM_DEVICE` must be set properly when you use the DRM backend. The default value is `/dev/dri/card0`.

Reply:
I fixed it.